.travis.yml

```diff
@@ -44,10 +33,10 @@ before_script:
   - sudo rm -f /etc/boto.cfg
 
 script:
-  - make test lint
+  - tox
 
 after_success:
-  if [ "$COVER" = "1" ]; then coveralls -v ; fi
+  coveralls -v
```

Comment:
what would happen here, given that each step in the matrix would execute this? Is coveralls smart enough to merge coverage?

Reply:
yes it is.
But I think coverage may not be running now (when running pytest)..
still trying to figure out how to make sure it is enabled.
will push an update in the next few hours.

Reply:
![image](https://user-images.githubusercontent.com/3523016/60770650-6e2ff800-a0ab-11e9-9168-5e0c1a6efbd8.png)

Reply:
🤦‍♂ fixing

Reply:
https://coveralls.io/builds/24412824